test/test_utils.py

```diff
@@ -488,6 +489,20 @@ def test_list(self):
         assert sanitize_list_for_csv([1, ["ab", "=de"]]) == [1, ["ab", "'=de"]]
 
 
+class TestValidateURL:
```

Comment:
Should probably mark as flaky if for whatever reason these urls are unavailable

Reply:
Good catch, thanks @freddyaboulton!